.github/workflows/tests.yaml

```diff
@@ -68,3 +68,4 @@ jobs:
             . "$MINIFORGE_INSTALL_DIR/bin/activate" testing
             cd test
             python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw .
+            python examples/wave-eager.py
```

Comment:
We should really run *all* examples, as a separate job. (cf #18 )

Reply:
There is only one example currently (?). I separated that out as test step for now until we decide how to proceed with #18.